assets/colors/colors.ts

```diff
@@ -1,5 +1,5 @@
 export const colors = {
```

Comment:
Constants should never be named in lowercase. See https://google.github.io/styleguide/jsguide.html#naming-rules-by-identifier-type. All constants should be renamed